packages/components/src/internal/components/domainproperties/samples/SampleTypePropertiesPanel.tsx

```diff
@@ -338,6 +356,29 @@ class SampleTypePropertiesPanelImpl extends React.PureComponent<
                         </Col>
                     </Row>
                 )}
+
+                {allowTimepointProperties && !appPropertiesOnly && (
+                    <Row className="margin-top">
+                        <Col xs={2}> Auto-Link Data to Study </Col>
+                        <Col xs={5}>
+                            <FormControl
+                                componentClass="select"
+                                id={ENTITY_FORM_IDS.AUTO_LINK_TARGET}
+                                onChange={this.onFormChange}
+                                value={model.autoLinkTargetContainerId || ''}
+                            >
+                                <option key="_empty" value={null} />
+                                {containers &&
+                                    containers.map((container, i) => (
+                                        <option key={i} value={container.id}>
+                                            {container.name}
```

Comment:
See attached screenshot. I think we should be using the `container.path` for the display value in the select options since it is possible for multiple container names to be the same if they are in different paths.
<img width="789" alt="Screen Shot 2021-03-17 at 9 30 58 AM" src="https://user-images.githubusercontent.com/6411206/111486457-82901900-8705-11eb-8857-d4407d04f5b7.png">

Reply:
(I'm assuming that this change would be desired on the Assay side as well, since I've pulled the dropdown into a shared component)